proto/iot.proto

```diff
@@ -119,6 +119,31 @@ message Thermostat {
     Double cooling_setpoint = 11;
 }
 
+message ThermostatSchedule {
+    //Map day of week to daily schedule
+    map<string, ScheduleDay> scheduleMap = 1;
+}
+
+message ThermostatScheduleDay {
+    //Daily schedule is Multiple Blocks
+    repeated ScheduleBlock blocks = 1;
+}
+
+message ThermostatScheduleBlock {
+    //heating setpoint
+    //unit: celsius
+    Double heating_setpoint = 1;
+    //cooling setpoint
+    //unit: celsius
+    Double cooling_setpoint = 2;
+    //Current system mode of thermostat
+    //unit: xbos/iot/HVACMode
+    HVACMode mode = 3;
+    //Time of schedule retrieval
+    //format: RFC
+    string time = 4;
```

Comment:
@john-b-yang What does "schedule retrieval" mean? The time the schedule was downloaded? if so, where are we storing the time of day that the schedule is intended to take effect?

Reply:
Oops! That should not be schedule retrieval. That time refers to when the settings for that block are enforced. I will update the description, sorry that was an error on my part.

Reply:
Aka, this is the time of day that the schedule is intended to take effect!